superset-frontend/src/components/FormLabel/FormLabel.test.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { render, screen } from 'spec/helpers/testing-library';
+import { FormControl } from 'react-bootstrap';
+import FormLabel from './index';
+
+const mockedProps = {
+  children: 'Form label',
+  required: false,
+  htmlFor: 'form-control',
+};
+
+test('should render', () => {
+  const { container } = render(<FormLabel {...mockedProps} />);
+  expect(container).toBeInTheDocument();
+});
+
+test('should render a Label', () => {
+  render(
+    <>
+      <FormLabel {...mockedProps} />
+      <FormControl id="form-control" />
+    </>,
+  );
+  expect(screen.getByLabelText('Form label')).toBeInTheDocument();
+});
+
+test('should be shown as required', () => {
+  const requiredProps = {
+    ...mockedProps,
+    required: true,
+  };
+  render(<FormLabel {...requiredProps} />);
+  expect(screen.getByTestId('span-required')).toBeInTheDocument();
```

Comment:
I think we can replace the ID with getByText and parentNode.
`expect(screen.getByText('*').parentNode). toBeInTheDocument();`